### DISCUSSION TITLE
Antismash v7 release?

### DISCUSSION THREAD

**Discussion by AshSudarshan** [antismash/antismash#527] [General]
Any updates or timeline when Antismash v7 command line will come out? Thank you so much for this tool!!

**Comment by kblin** [comment]
Our usual timeline is that we release the downloadable version once the corresponding paper is accepted, as we want to make sure the features discussed in the paper and the features in the release are in sync. So assuming things go as they usually do we're looking at early spring?

**Reply by AshSudarshan** [reply]
Awesome. Thank you so much!

**Comment by snayfach** [comment]
Does that mean that the master branch currently contains all the features and functionality expected in v7, or is it still under active development?

**Comment by SJShaw** [comment]
We're not planning any significant changes between current master and the 7.0 release. However, there may be unplanned changes and minor fixes/tweaks before the 7.0 release.

**Comment by jolespin** [comment]
Will v7 include plantSMASH?

**Comment by SJShaw** [comment]
No, plantiSMASH is an independently operated and maintained fork.
